docs/UserGuide.md

```diff
@@ -120,7 +120,7 @@ The available dashboards are detailed below.
 By default, FriendDex displays your upcoming events on the details panel, such as upcoming birthdays and special dates.
 
 #### Streaks Dashboard
-The Streaks dashboard shows how consistent you stick to your goal of meeting your friends up till the current day.    
+The Streaks dashboard shows how consistent you stick to your goal of meeting your friends up till the current day. More information can be found [here](#streaks).
```

Comment:
copywriting nit: "The Streaks dashboard shows the number of consecutive times you've met your goals of meeting your friends up till today. More information on Streaks is available [here](#streaks)."

Reply:
I will leave the explicit consecutive times out because technically any valid meetings will still add on to the streak if it is in the correct time period.